### PR TITLE
Conda recipe changes for py3.9 and better compatibility with WESTPA 2.0

### PR DESCRIPTION
**Pull request by jeremyleung521**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jeremyleung521** [comment]
I had to move the install path back to ${PREFIX} because of the wonkiness during the `conda-build`'s testing modules. They move the source folder around during the process when it creates a new environment to test, breaking links to python and such. 

It would be nice to a have a WESTPA 1.0 installation per-environment but it's completely incompatible with the current testing. I tried, but it failed at the point calling a suitable python. It probably requires rewriting the westpa code where it that defines $WEST_PYTHON, because it seems like conda-build installs it then moves it for testing. Anyhow this will all be moot when WESTPA 2.0 releases.

**Comment by jeremyleung521** [comment]
@synapticarbors Hoping you'll be able to take a look at this and merge it in soon! Thanks in advance.

**Comment by synapticarbors** [comment]
@conda-forge-admin please rerender

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: linux
   pool:
-    vmImage: ubuntu-16.04
```
Comment (synapticarbors) [inline .azure-pipelines/azure-pipelines-linux.yml:8]: @jeremyleung521 Can you explain this change? Typically we don't want to touch anything that's configured by conda-forge.

Comment (jeremyleung521) [reply .azure-pipelines/azure-pipelines-linux.yml:8]: Can't seem to find the exact error message line at the moment, but it's related to this: https://github.com/actions/virtual-environments/issues/3287 

I got a warning while I was testing earlier and thought i should also bundle the change in.

Comment (synapticarbors) [reply .azure-pipelines/azure-pipelines-linux.yml:8]: Ok, it actually looks like when I rerendered the feedstock that it wanted to make that change anyway, so I'm fine with it.

**Comment by synapticarbors** [comment]
closes #26

**Pull request merged**